1.4/Defs/ThingDefs/Emplacements_CEA/12Pounder.xml

```diff
@@ -27,7 +27,7 @@
         <warmupTime>3.3</warmupTime>
         <range>62</range>
         <minRange>14</minRange>
-        <soundCast>120mm</soundCast>
+        <soundCast>Shot_CE_Musket</soundCast>
```

Comment:
we definitely will need a sound effect for this
and a garand ping
in a later PR